tests/openssh.rs

```diff
@@ -376,62 +374,63 @@ macro_rules! assert_remote_kind {
 
 #[tokio::test]
 #[cfg_attr(not(ci), ignore)]
-async fn sftp_can() {
+#[cfg(feature = "scp")]
```

Comment:
These can go away now too!

Reply:
Thanks for reminding me!

The integration tests passed, so I forgot to remove them.